packages/api-client/src/views/Request/RequestSection/RequestBody.test.ts

```diff
@@ -64,7 +64,7 @@ describe('RequestBody.vue', () => {
 
   it('renders correctly with no body', async () => {
     const wrapper = mount(RequestBody, props)
-    expect(wrapper.text()).toContain('No Body')
+    expect(wrapper.findComponent({ name: 'ScalarListbox' }).text()).toContain('None')
```

Comment:
wasn't able to test this issue but these are a bit better so they look at the listbox